tests/gnu/testlet/MIDletTestlet.java

```diff
@@ -0,0 +1,4 @@
+package gnu.testlet;
+
+public interface MIDletTestlet extends Testlet {
```

Comment:
I added this interface to be able to simply grep to find classes to add to the MIDletTestlets list.